src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,64 @@
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+// const DrcHelpers = require('../utils/helpers.js');
+// const ENVIRONMENTS = require('site/constants/environments');
+// const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
+const inputs = require('./utils/inputs.json');
+const DrcE2eHelpers = require('./drc-helpers.js');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  // Navigate to beta-page.
+  client
+    .openUrl(`${E2eHelpers.baseUrl}/disability/about-disability-ratings-beta/`)
```

Comment:
Looks like the Jenkins check failed at the page-title assertion (got "Error" instead of the expected page-title)...

Is our beta page accessible, wherever Jenkins is checking this?

The beta page comes up fine, and the page-title assertion passes, on my local.

Reply:
Added:
`module.exports['@disabled'] =
  process.env.BUILDTYPE === ENVIRONMENTS.VAGOVPROD &&
  drcPagePath.contains('-beta');`